Content.Client/GameObjects/Components/StationEvents/RadiationPulseComponent.cs

```diff
@@ -9,31 +9,17 @@ namespace Content.Client.GameObjects.Components.StationEvents
     [ComponentReference(typeof(SharedRadiationPulseComponent))]
     public sealed class RadiationPulseComponent : SharedRadiationPulseComponent
     {
-        private float _dps;
+        private bool _draw;
+        private bool _decay;
+        private float _radsPerSecond;
         private float _range;
         private TimeSpan _endTime;
 
-        public override float DPS
-        {
-            get => _dps;
-            set
-            {
-                _dps = value;
-                Dirty();
-            }
-        }
-
-        public override float Range
-        {
-            get => _range;
-            set
-            {
-                _range = value;
-                Dirty();
-            }
-        }
```

Comment:
Removing this means that changing the values on the client may potentially put the radiation pulse out of sync with the server.

Reply:
You shouldn't be able to modify these values on the client at all